vivarium/processes/timeline.py

```diff
@@ -8,6 +8,15 @@
 from vivarium.core.process import Process, Generator
 
 
+
+def nested_set(dic, keys, value):
+    ''' make an embedded dict from a list of keys,
+    with a value set at the deepest key '''
```

Comment:
The first line of a docstring should stand on its own. This is what the docs will use as the short description of the function. Other lines should follow after a blank line.